docs/references/strategies/languages/gradle/gradle.md

```diff
@@ -282,6 +283,68 @@ We classify following configurations, and dependencies originating from it as a
 - kotlinNativeCompilerPluginClasspath
 ```
 
+## Only analyzing specific sub project
+
+If you have gradle project which has one or more subprojects, you may
+only want to analyze a specific set of subprojects in some cases.
+
+
+In `fossa-cli`, this can be achieved by using [exclusion filtering](./../../../files/fossa-yml.md).
+
+1) Run `fossa list-targets`, to identify project directory and identifier of subprojects.
+```bash
+[ INFO] Found project: gradle@./
+[ INFO] Found target: gradle@./::app
+[ INFO] Found target: gradle@./::list
+[ INFO] Found target: gradle@./::utilities
+```
+
+Note that, targets are denoted in following format `type@path:target`. For 
+example `gradle@./::utilities`:
+
+```
+gradle  @           ./      :         :utilities
```

Comment:
We should explicitly call out the extra `:` here as coming from gradle.  I think with that example and the comment, we are fully communicating what we need to.  Something like `gradle attaches leading colons to submodules, so the utilities submodule here is referenced by ":utilities"`.  Probablybest placed above this visual breakdown.